Makefile

```diff
@@ -186,7 +182,7 @@ BUILD = GOOS=$(1) GOARCH=$(2) \
 	-ldflags="$(LD_FLAGS)" \
 	-gcflags="$(GC_FLAGS)" \
 	-asmflags="$(ASM_FLAGS)" \
```

Comment:
As above, maybe this `-asmflags=...` like could dropped now, since it seems like it was added along with `ASM_FLAGS` for this one purpose.